rpcs3/Emu/NP/rpcn_client.cpp

```diff
@@ -1590,11 +1590,14 @@ namespace rpcn
 		return true;
 	}
 
-	bool rpcn_client::req_ticket(u32 req_id, const std::string& service_id)
+	bool rpcn_client::req_ticket(u32 req_id, const std::string& service_id, const std::vector<u8>& cookie)
 	{
 		std::vector<u8> data;
 		std::copy(service_id.begin(), service_id.end(), std::back_inserter(data));
 		data.push_back(0);
+		const le_t<u32> size = cookie.size();
+		std::copy(&size, &size + sizeof(le_t<u32>), std::back_inserter(data));
```

Comment:
There must be a bug here, this isn't std::fill.

Reply:
better now? :)

Reply:
I mean, I do not see how &size + sizeof(le_t<u32>) - 1 is a valid memory location to address.

Reply:
It's not but that's why I changed the code to
```cpp
std::copy(reinterpret_cast<const u8 *>(&size), reinterpret_cast<const u8 *>(&size) + sizeof(le_t<u32>), std::back_inserter(data));
```

Reply:
Ah ok. 👍